life_expectancy/cleaning.py

```diff
@@ -98,12 +165,20 @@ def main(region: str ="PT") -> None:
         '--region',
         type=str,
         help="Indicate preferred country with 2 capital letters (default= 'PT').",
-        default="PT",
+        default=Country.PT,
+    )
+    parser.add_argument(
+        '-f',
+        '--format_type',
+        type=str,
+        help="Select source file format. Default is csv, may also choose json",
+        default="csv",
```

Comment:
In my opinion, it would be better to use a method that recognizes the file extension [eg. split()] rather than manually specifying it.

Reply:
It depends on the use case though. I mean, what if there isn't a file extension (e.g. an API ou an database connection)?